docs/capn-tutorial.md

```diff
@@ -0,0 +1,110 @@
+## Quick Start 
+
+This tutorial introduces how to create a nested controlplane. . CAPN should work with any standard 
+Kubernetes cluster out of box, but for demo purposes, in this tutorial, we will use 
+a kind cluster as the meta cluster as well as the infrastructure provider.
+
+### Prerequests
+
+Please install the latest version of [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) 
+and [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+### Clone CAPN
+
+```shell
+git clone https://github.com/kubernetes-sigs/cluster-api-provider-nested
+cd cluster-api-provider-nested
+```
+
+### Create `kind` cluster
+
+```shell
+kind create cluster
+```
+
+### Install `cert-manager`
+
+**TODO: briefly explain why install cert-manager** 
+
+```shell
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
+```
+
+### Clone CAPI and Deploy Dev release
+
+Though CAPN can be used independently, as a cluster API~(CAPI) provider, 
```

Comment:
it can't actually be independently deployed without hacking a bunch of the connection between `Cluster` and `NestedCluster` which sets up the initial ownership. I'm not sure how best to phrase that.